src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.cs

```diff
@@ -158,8 +158,10 @@ private static ImmutableArray<Alias> GetAliases(DkmClrRuntimeInstance runtimeIns
                     (blocks, useReferencedModulesOnly) => CreateMethodContext(instructionAddress, blocks, useReferencedModulesOnly),
                     (context, diagnostics) =>
                     {
+                        Debug.Assert(lValue.FullName is not null);
+
                         var compileResult = context.CompileAssignment(
-                            lValue.FullName,
+                            lValue.FullName!, // Concord marks this as nullable but it should always have a value in our scenario.
```

Comment:
>! [](http://example.com/codeflow?start=43&length=1)

If the compiler requires this `!` you should be able to use `RoslynDebug.Assert` above instead.